src/features/session/login/login-btn.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { onLogin } from "entities/session";
+import { Button } from "shared/ui";
+
+const LoginButton = () => {
+  return (
+    <Button
+      onClick={() => onLogin()}
+      sx={{ fontSize: "18px", px: 4, width: "300px" }}
+      variant="contained"
+    >
+      Sign In
+    </Button>
+  );
+};
+
+export default LoginButton;
```

Comment:
we dont use default export

Reply:
so than how to deal with such an error?) 
![image](https://user-images.githubusercontent.com/87814580/223049015-27631a9a-96f7-4f60-b476-0df17532c7c1.png)

Reply:
got it. changed